TODO

```diff
@@ -0,0 +1,14 @@
+Todo
+
+1. Generate sample document for
+    - direct signed by eth (DID)
+    - direct signed by eth and using dns proof (DNS-DID)
```

Comment:
DNS-DID or DID-DNS ? I see both used

Reply:
the first word is the top level identifier.... so
DNS-DID
DID
DNS-CONTRACT (replacing DNS-TXT)
DID-CONTRACT